app/views/hyrax/base/_items.html.erb

```diff
@@ -5,7 +5,9 @@
 
 <%# BUTTON: Add in the deletion of all files %>
 <div class="btn-line">
-  <%= link_to "Delete All Files", polymorphic_path([main_app, :destroy_all_files, :hyrax, presenter.model_name.name.downcase.to_sym]), class: 'btn btn-danger', data: { confirm: "Are you sure you want to delete all files?" }, disabled: presenter.file_set_presenters.blank?, method: :delete %>
+  <% if presenter.file_set_presenters.blank? == false %>
```

Comment:
This works but instead of a comparison like this we can just do `if !presenter.file_set_presenters.blank?`

If it is blank itll return true, but then make it false so it wont fulfill the if statement.

If it isnt blank itll return false but then make the false true so itll fulfill the if statement.